src/Symfony/Component/Workflow/WorkflowEvents.php

```diff
@@ -55,4 +55,9 @@ final class WorkflowEvents
     private function __construct()
     {
     }
+
+    public static function getDefaultDispatchEvents(): array
```

Comment:
```suggestion
    public static function getDefaultDispatchedEvents(): array
```

if you agree to rewrite the property above